package.json

```diff
@@ -48,7 +48,7 @@
     "docker-progress": "^3.0.1",
     "dockerode": "^2.5.5",
     "lodash": "^4.17.4",
-    "resin-bundle-resolve": "^2.0.0",
+    "resin-bundle-resolve": "^2.0.1",
```

Comment:
This change requires balena-io-modules/resin-bundle-resolve/pull/29 to be merged first